src/core/router/routes.tsx

```diff
@@ -27,6 +29,54 @@ const { BivariateManagerPage } = lazily(
 export const isAuthenticated = !!configRepo.get().user;
 export const isMapFeatureEnabled = configRepo.get().features[AppFeature.MAP];
 
+const ABOUT_SUBTABS: Record<string, Omit<AppRoute, 'view' | 'parentRouteId'>> = {
+  terms: {
+    id: 'terms',
+    slug: 'terms',
+    title: i18n.t('modes.terms'),
+    icon: <Reports16 />,
+    visibilityInNavigation: 'always',
+    requiredFeature: AppFeature.ABOUT_PAGE,
+  },
+  privacy: {
+    id: 'privacy',
+    slug: 'privacy',
+    title: i18n.t('modes.privacy'),
+    icon: <Reports16 />,
+    visibilityInNavigation: 'always',
+    requiredFeature: AppFeature.ABOUT_PAGE,
+  },
+  'user-guide': {
+    id: 'user-guide',
+    slug: 'user-guide',
+    title: i18n.t('modes.user_guide'),
+    icon: <BookOpen24 />,
+    visibilityInNavigation: 'always',
+    requiredFeature: AppFeature.ABOUT_PAGE,
+  },
+};
+
+function getAboutSubTabs() {
+  const subTabsConfig = configRepo?.get().features.about_page?.['subTabs'] as
+    | AboutFeatureConfig['subTabs']
+    | undefined;
+  if (Array.isArray(subTabsConfig)) {
+    return subTabsConfig
+      .filter((subTab) => ABOUT_SUBTABS[subTab.tabId] && subTab.assetUrl)
```

Comment:
If `assetUrl `optional then make appropriate changes to the type

Reply:
Fixed